docs/develop/dialog-best-practices.md

```diff
@@ -18,7 +18,7 @@ This article provides rules, gotchas, and best practices for the Office dialog A
 
 - The dialog box can only navigate to HTTPS URLs, not HTTP.
 - The URL passed to the [displayDialogAsync](/javascript/api/office/office.ui) method must be in the exact same domain as the add-in itself. It cannot be a subdomain. But the page that is passed to it can redirect to a page in another domain.
-- A host window, which can be a task pane or the UI-less [function file](/javascript/api/manifest/functionfile) of an add-in command, can have only one dialog box open at a time.
+- A host page, which can be a task pane or the [function file](/javascript/api/manifest/functionfile) of a [function command](../design/add-in-commands.md#types-of-add-in-commands), can have only one dialog box open at a time.
```

Comment:
Unrelated, but the definition seems to hide the limitation. Consider this reframing.
```suggestion
- A host page can have only one dialog box open at a time. The host page could be either a task pane or the [function file](/javascript/api/manifest/functionfile) of a [function command](../design/add-in-commands.md#types-of-add-in-commands).
```